commit 7b09d79de3734a79896995c6b71e2005618bfeac (refs/changes/91/3191/2)
Author: David Pacheco <dap@joyent.com>
Date:   2018-01-10T15:36:47-08:00 (1 year, 9 months ago)
    
    joyent/manta#17 Typo - MUSKIE_MULTI_DC
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Reviewed by: Tim Kordas <tim.kordas@joyent.com>
    Approved by: Brian Bennett <brian.bennett@joyent.com>

diff --git a/docs/operator-guide.md b/docs/operator-guide.md
index 8372784..323d7e4 100644
--- a/docs/operator-guide.md
+++ b/docs/operator-guide.md
@@ -847,7 +847,7 @@ multi-DC, multi-compute-node deployment.  The general process is:
     This is required to enforce that object writes are distributed to multiple
     datacenters.  In the SAPI master datacenter:
 
-        headnode $ app_uuid="$(sdc-sapi /applications?name=manta | json -H uuid)"
+        headnode $ app_uuid="$(sdc-sapi /applications?name=manta | json -Ha uuid)"
         headnode $ echo '{ "metadata": { "MUSKIE_MULTI_DC": true } }' | \
             sapiadm update "$app_uuid"
 
